README.md

```diff
@@ -256,6 +256,14 @@ closes **go-jira** will submit the completed form.  The order which **go-jira**
 * **EDITOR** environment variable
 * vim
 
+To automate editing, use the `--noedit` to skip opening the editor, and pass `--override` fields as defined in `~/.jira.d/templates/edit`: 
+
+```
+jira edit GOJIRA-321 --noedit --override priority=P0
```

Comment:
@darthwalsh How do you override nested/arrays fields e.g. if you wanted to add `Component2` below to the jira below

```yaml
fields:

  components:
    - name: Component1
    - name: Component2
```

Reply:
I'm not sure! If you open `~/.jira.d/templates/edit` there's some complicated templating logic that looks like it merges together `.overrides.components` with existing `.fields.components`? But you'd have to try to be sure.